src/plugins/discover/public/context_awareness/profile_providers/profile_provider_services.ts

```diff
@@ -8,21 +8,22 @@
  */
 
 import { createLogsContextService, LogsContextService } from '@kbn/discover-utils';
+import type { LogsDataAccessPluginStart } from '@kbn/logs-data-access-plugin/public';
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface ProfileProviderDeps {
-  // We will probably soon add uiSettings as a dependency
-  // to consume user configured indices
+  logsDataAccessPlugin?: LogsDataAccessPluginStart;
 }
 
 export interface ProfileProviderServices {
   logsContextService: LogsContextService;
 }
 
-export const createProfileProviderServices = (
-  _deps: ProfileProviderDeps = {}
-): ProfileProviderServices => {
+export const createProfileProviderServices = async (
+  _deps: ProfileProviderDeps
```

Comment:
```suggestion
  deps: ProfileProviderDeps
```
Nit: we could remove `_` now that it's used.